internal/store/store.go

```diff
@@ -138,3 +141,60 @@ func (db *DB) RunMigrations(dir string) error {
 
 	return nil
 }
+
+func CreateSourceFromEbmedded(assets embed.FS, dir string, versions []int) (sources.Source, error) {
```

Comment:
typo: `CreateSourceFromEmbedded`

Also, I think this could be a functionality within morph itself which we could just import? IIRC, there was already something similar in morph.

Reply:
Yes but we are filtering the versions here